docs/topics/readme/cd-circuit-breaker-spring-boot.properties

```diff
@@ -5,5 +5,6 @@ fileLocation=src/main/resources/static/index.html
 greeting-app-name=springboot-cb-greeting
 name-app-name=springboot-cb-name
 gitCMD=git clone git@github.com:${loggedUser}/${targetRepository}
-cicdSection=IMPORTANT: As part of the process of creating this booster, developers.redhat.com/launch or the Fabric8 Launcher tool set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster or OpenShift Online Web Console. 
-prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x or greater.
\ No newline at end of file
+cicdSection=IMPORTANT: As part of the process of creating this booster, developers.redhat.com/launch or the Fabric8 Launcher tool set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster or OpenShift Online Web Console.
+prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x or greater.
+integrationTestSection=NOTE: Run the set of integration tests included with this booster using `$ mvn clean verify -Popenshift,openshift-it`.
```

Comment:
I understand that the usage of `$` is meant to signify a shell prompt, but I think that only makes sense in larger code blocks. When you include a command inline into a sentence, I think including the prompt marker does more harm than good.

Reply:
I concur, we typically include the shell prompt only in code blocks (as opposed to inline code) because in sentences, you can easily say "as a regular user" or "as root" and it's natually part of the information.

Reply:
@Ladicek @tradej Thank you for the feedback. I agree, including the shell prompt does not really make sense for an inline command example like this. Removing...